src/pg_to_evalscript/node.py

```diff
@@ -245,3 +246,28 @@ def write_process(self):
     return newData;
 }}
 """
+
+
+class CountNode(Node):
+    def is_process_defined(self, process_id):
+        return True
+
+    def write_process(self):
+        newline = "\n"
+        tab = "\t"
+        count_process_definition = self.load_process_code()
+        # remove first and last line from file, so we get only content of count process (function)
+        count_process_definition_content = count_process_definition[count_process_definition.find(newline) + 1 : count_process_definition.rfind('}')]
```

Comment:
I think this should not be necessary, you should be able to call the `count` function from inside the wrapper `count` function afaik.